torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -61,6 +61,7 @@ const std::unordered_map<Symbol, Symbol>& getOperatorAliasMap() {
       {aten::arctanh_, aten::atanh_},
       {aten::fix, aten::trunc},
       {aten::fix_, aten::trunc_},
+      {aten::positive, aten::clone},
```

Comment:
It's probably simpler to remove the alias for now and update the OpInfo to just be positive. See comment in common_methods_invocations.py for more information on this approach.